src/routes/families.ts

```
@@ -47,16 +47,22 @@ familyRouter.put(
       const detId = parseInt(req.params['determinationId']);
       const readOrgIds = await getReadAccessibleOrgIds(req.user);
 
-      const detToModify = await getManager().findOne(
-        IncomeDetermination,
-        {
-          id: detId,
-          familyId: famId,
+      // Need to build out the options dict because we want to do property
+      // filtering on a nested relation alias
+      const opts: FindOneOptions<IncomeDetermination> = {
```

an option here is to do what we do in the enrollment/funding controller: 
first try to get the appropriate family
If the family with given orgId doesn't exist, throw not found
Then if the family exists, grab the income determination
 If determination with given family id doesn't exist throw not found

An extra query, but perhaps worth it for readability since you can then just use vanilla typeorm and not have to do the alias-based manual wheres

---

I find the alias-based where to be a huge pain to read so if the extra query is okay I would def prefer to have that in there